src/components/_global/BalAsset/BalAssetSet.vue

```diff
@@ -55,3 +56,16 @@ export default defineComponent({
   }
 });
 </script>
+
+<style scoped>
+.token-set {
+  @apply absolute;
+  box-shadow: 0 0 0 1.5px #ffffff;
+  border-radius: 50%;
```

Comment:
I don't think this is needed?

Reply:
The border-radius is only really needed for the unknown token symbol which actually appears to be a square (which also randomly appears to be larger than the other tokens...). Because it's a square, the border-radius is required if we want to do the border separation technique.

Reply:
Ahh ok, yeah the child components handle this as far as I'm aware with the `rounded-full` class but if we need it at this level too then let's use that.

Reply:
Done